spring-boot-project/spring-boot-autoconfigure/pom.xml

```diff
@@ -790,6 +790,16 @@
 			<artifactId>neo4j-ogm-embedded-driver</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
dependencies are added in natural order.

Reply:
I don't understand that one. What is the natural order of the dependencies?

Reply:
alphabetical order (`groupId` then `artifactId`).